src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const numbersArray = [...document.querySelectorAll('.population')];
+let tot = 0;
+
+numbersArray.forEach((el) => (tot += parseFloat(el.innerText.replace(/[^0-9.-]+/g, ''))));
+
+const average = tot / numbersArray.length;
```

Comment:
Ensure that `numbersArray.length` is not zero to avoid division by zero, which would result in an Infinity value for the average.